C/tests/c4DatabaseTest.cc

```diff
@@ -1075,7 +1075,9 @@ TEST_CASE("Database Upgrade From 2.7", "[Database][Upgrade][C]") {
 }
 
 
-TEST_CASE("Database Upgrade From 2.7 to Version Vectors", "[Database][Upgrade][C]") {
+TEST_CASE("Database Upgrade From 2.7 to Version Vectors", "[Database][Upgrade][C][.failed]") {
+    auto dbDomain = c4log_getDomain("DB", false);
+    c4log_setLevel(dbDomain, kC4LogDebug);
```

Comment:
Can we remove this?  It will cause log spam in normal cases.

Reply:
Oh, I forgot to remove that.